internal/cmd/kafka/command_cluster_describe.go

```diff
@@ -101,8 +102,8 @@ func (c *clusterCommand) outputKafkaClusterDescription(cmd *cobra.Command, clust
 
 	if getTopicCount {
 		topicCount, err := c.getTopicCountForKafkaCluster(cluster)
-		if err != nil {
-			return err
+		if err != nil { // topicCount is 0 when err != nil, and will be omitted by `omitempty`
```

Comment:
It's a better practice to put comments above the line they refer too, this prevents lines from getting too long.
```suggestion
		// topicCount is 0 when err != nil, and will be omitted by `omitempty`
		if err != nil {
```